control-plane/stor-port/src/types/v0/store/replica.rs

```diff
@@ -108,6 +109,14 @@ impl ReplicaSpec {
     pub fn owned_by(&self, id: &VolumeId) -> bool {
         self.owners.owned_by(id)
     }
+    /// Check if this replica is owned by the nexus.
+    pub fn owned_by_nexus(&self, nexus_id: Option<&NexusId>) -> bool {
```

Comment:
Should probably be a list of nexus_id's which are retrieved from volume, specs.volume_nexuses IIRC